Tasks/VSMobileCenterTest/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -1,6 +1,6 @@
 {
   "loc.friendlyName": "Mobile Center Test",
-  "loc.helpMarkDown": "Test mobile app packages with Visual Studio Mobile Center.",
+  "loc.helpMarkDown": "[Support Information](https://intercom.help/mobile-center/)",
```

Comment:
I think it would be better to append the support link to the help markdown. @DavidStaheli will be able to help with better wording.